sigmastate/src/main/scala/sigmastate/serialization/ModQSerializer.scala

```diff
@@ -5,7 +5,7 @@ import sigmastate.lang.Terms._
 import sigmastate.utils.{SigmaByteReader, SigmaByteWriter}
 import sigmastate.{ModQ, SType}
 
-// TODO HF (2h): make sure it is covered with tests
+// TODO v6.0 (2h): make sure it is covered with tests
```

Comment:
This class doesn't make sense before v6.0. It cannot be removed however, because this serializer was erroneously registered in ValueSerializer's global map. See comment [here](https://github.com/ScorexFoundation/sigmastate-interpreter/blob/dac1109d284b5342859fcd4a0b34b87a502a029d/sigmastate/src/main/scala/sigmastate/serialization/ValueSerializer.scala#L143)